src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -16051,5 +16218,4893 @@ struct C2 : I1<C2>
+                Assert.False(cM01Set.IsAbstract);
+                Assert.False(cM01Set.IsVirtual);
+                Assert.False(cM01Set.IsMetadataVirtual());
+                Assert.False(cM01Set.IsMetadataFinal);
+                Assert.False(cM01Set.IsMetadataNewSlot());
+                Assert.Equal(MethodKind.PropertySet, cM01Set.MethodKind);
+                Assert.False(cM01Set.HasRuntimeSpecialName);
+                Assert.True(cM01Set.HasSpecialName);
+
+                Assert.Equal("void C.I1.M01.set", cM01Set.ToTestDisplayString());
+
+                Assert.Same(m01, cM01.ExplicitInterfaceImplementations.Single());
+                Assert.Same(m01Get, cM01Get.ExplicitInterfaceImplementations.Single());
+                Assert.Same(m01Set, cM01Set.ExplicitInterfaceImplementations.Single());
+            }
+        }
+
+        [Fact]
+        public void ImplementAbstractStaticProperty_09()
+        {
+            // Explicit implementation from base is treated as an implementation
+
+            var source1 =
+@"
+public interface I1
+{
+    abstract static int M01 { get; set; }
+}
+
+public class C1
+{
+    public static void M01() {}
+}
+
+public class C2 : C1, I1
+{
+    static int I1.M01 { get; set; }
+}
+";
+
+            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.RegularPreview,
+                                                 targetFramework: TargetFramework.NetCoreApp);
+
+            compilation1.VerifyDiagnostics();
+
+            var source2 =
+@"
+public class C3 : C2, I1
+{
+}
+";
+
+            foreach (var reference in new[] { compilation1.ToMetadataReference(), compilation1.EmitToImageReference() })
+            {
+                foreach (var parseOptions in new[] { TestOptions.Regular9, TestOptions.RegularPreview })
+                {
+                    var compilation2 = CreateCompilation(source2, options: TestOptions.DebugDll,
+                                                     parseOptions: parseOptions,
+                                                     targetFramework: TargetFramework.NetCoreApp,
+                                                     references: new[] { reference });
+                    CompileAndVerify(compilation2, sourceSymbolValidator: validate, symbolValidator: validate, verify: Verification.Skipped).VerifyDiagnostics();
+                }
+            }
+
+            void validate(ModuleSymbol module)
+            {
+                var c3 = module.GlobalNamespace.GetTypeMember("C3");
+                Assert.Empty(c3.GetMembers().OfType<PropertySymbol>());
+                Assert.Empty(c3.GetMembers().OfType<MethodSymbol>().Where(m => !m.IsConstructor()));
+                var m01 = c3.Interfaces().Single().GetMembers().OfType<PropertySymbol>().Single();
+
+                var cM01 = (PropertySymbol)c3.FindImplementationForInterfaceMember(m01);
```

Comment:
nit: I feel the test would read a little more easily if the name were `c3M01` rather than `cM01`